both/routes.js

```diff
@@ -285,6 +285,9 @@ Router.route('/manage/:competitionUrlId/events', {
   name: 'editEvents',
   controller: 'ManageCompetitionController',
   titlePrefix: "Edit events",
+  waitOn: function() {
+    return [subs.subscribe('roundProgresses', this.params.competitionUrlId, subscriptionError(this))];
```

Comment:
Nice! A good side effect of pulling the progresses out is that the server will have less data it needs to push out to clients who are subscribed to rounds, but don't care about the rounds progresses.